djstripe/webhooks.py

```diff
@@ -39,9 +40,10 @@
 
 def handler(event_types):
     """
-    Decorator which registers a function as a webhook handler for the specified
-    event types (e.g. 'customer') or fully qualified event sub-types (e.g.
-    'customer.subscription.deleted').
+    Decorator that registers a function as a webhook handler.
+
+    Is used for the specified event types (e.g. 'customer') or fully qualified event sub-types
+    (e.g. 'customer.subscription.deleted').
 
     If an event type is specified then the handler will receive callbacks for
```

Comment:
Can you change "specified then" to just "specified, " in both instances in this paragraph?